metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/MathFunctions.java

```diff
@@ -19,45 +19,179 @@
     }
 
-    @Override
-    public void initialize(Context context) {
+  }
 
+  @Stellar(name="SQRT"
+          ,description="Returns the square root of a number."
+          ,params = {
+                "number - The number to take the square root of"
+                    }
+          , returns="The square root of the number passed in."
+          )
+  public static class Sqrt extends StellarMathFunction {
+   public Sqrt() {
+      super(MathOperations.SQRT);
     }
 
-    @Override
-    public boolean isInitialized() {
-      return true;
+  }
+
+  @Stellar(name="CEIL"
+          ,description="Returns the ceiling of a number."
+          ,params = {
+                "number - The number to take the ceiling of"
+                    }
+          , returns="The ceiling of the number passed in."
+          )
+  public static class Ceil extends StellarMathFunction {
+   public Ceil() {
+      super(MathOperations.CEIL);
     }
+
   }
 
+  @Stellar(name="FLOOR"
+          ,description="Returns the floor of a number."
+          ,params = {
+                "number - The number to take the floor of"
+                    }
+          , returns="The floor of the number passed in."
+          )
+  public static class Floor extends StellarMathFunction {
+   public Floor() {
+      super(MathOperations.FLOOR);
+    }
+  }
+
+  @Stellar(name="SIN"
+          ,description="Returns the sin of a number."
+          ,params = {
+                "number - The number to take the sin of"
+                    }
+          , returns="The sin of the number passed in."
+          )
+  public static class Sin extends StellarMathFunction {
+   public Sin() {
+      super(MathOperations.SIN);
+    }
+  }
+
+  @Stellar(name="COS"
+          ,description="Returns the cos of a number."
+          ,params = {
+                "number - The number to take the cos of"
+                    }
+          , returns="The cos of the number passed in."
+          )
+  public static class Cos extends StellarMathFunction {
+   public Cos() {
+      super(MathOperations.COS);
+    }
+  }
+
+  @Stellar(name="TAN"
+          ,description="Returns the tan of a number."
+          ,params = {
+                "number - The number to take the tan of"
+                    }
+          , returns="The tan of the number passed in."
+          )
+  public static class Tan extends StellarMathFunction {
+   public Tan() {
+      super(MathOperations.TAN);
+    }
+  }
+
+  @Stellar(name="EXP"
+          ,description="Returns Euler's number raised to the power of the argument"
+          ,params = {
+                "number - The power to which e is raised."
+                    }
+          , returns="Euler's number raised to the power of the argument."
+          )
+  public static class Exp extends StellarMathFunction {
+   public Exp() {
+      super(MathOperations.EXP);
+    }
+  }
+
+  @Stellar(name="ROUND"
+          ,description="Rounds a number to the nearest integer"
```

Comment:
Would you mind mentioning explicitly that this is half-up rounding?  That's going to be the default assumption, but it should be explicit since we aren't supporting other modes for right now.